tests/link/pytorch/test_nlinalg.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import pytest
+
+from pytensor.configdefaults import config
+from pytensor.graph import FunctionGraph
+from pytensor.graph.op import get_test_value
+from pytensor.tensor.math import argmax
+from pytensor.tensor.type import matrix
+from tests.link.pytorch.test_basic import compare_pytorch_and_py
+
+
+@pytest.mark.parametrize(
+    "keepdims",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "axis",
+    [None, 1, (0,)],
```

Comment:
Can you make the test tensor a tensor3 (3 dims), and test also `axis=(0, 2)`? This is something PyTensor supports but not e.g., numpy and perhaps pytorch as well. In that case you'll need to copy the logic used in `perform` instead of just calling `argmax`. If it works, great!

Reply:
In general with axis Ops we should always test `None`, (x,) and (x, y), for 3d inputs, so we test all, a single axis or partial axes